examples/nodejs/aws/eventbridge/webapp/src/components/InfoModal.tsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect } from "react";
+import { Help } from "../svgs/help";
+
+type InfoModalProps = {
+  isVisible: boolean;
+  onClose: () => void;
+};
+
+const appUsageInfo = `
+<strong>Application Overview:</strong><br>
+The application showcases a write-through cache pattern implementation for DynamoDB utilizing DynamoDB Streams, AWS EventBridge, and Momento. It enables users to perform create, update, and delete operations on items within a DynamoDB table, with real-time reflection of changes in the cache (<span class="italic">"momento-eventbridge-cache"</span>) and topic (<span class="italic">"momento-eventbridge-topic"</span>).<br><br>
+
+<strong>How to Use the App:</strong><br><br>
+<strong>Prerequisites:</strong><br>
+Before interacting with the application, ensure the existence of the cache <span class="italic">"momento-eventbridge-cache"</span>. If the cache does not exist, it can be created via <a class="text-blue-500" href="https://console.gomomento.com" target="_blank" rel="noopener noreferrer">Momento Console</a>.<br><br>
+
+<strong>Getting Started:</strong><br>
+1. <strong>Create Weather Record:</strong> Begin by creating a weather record within the DynamoDB table. The application does not enforce any constraints on the values entered, allowing users to input any desired data.<br>
+2. <strong>Retrieve Cache Item:</strong> After submitting the weather record, observe that the cache reflects the same contents as the DynamoDB table. This demonstrates the real-time synchronization between the two data sources.<br>
+3. <strong>Publish to Topic:</strong> Additionally, notice that a message is published to the topic, mirroring the contents of the DynamoDB table. This showcases the propagation of changes across subscribed topic.<br>
+4. <strong>Delete Record:</strong> Proceed to delete the previously created record from the DynamoDB table.<br>
+5. <strong>Check Cache:</strong> Upon attempting to retrieve the item from the cache post-deletion, expect a cache miss indication. This confirms the absence of the deleted item in the cache.<br><br>
```

Comment:
`Click the retrieve button again and observe that the item has been removed from both DynamoDB and the Momento Cache`